__tests__/integration.test.js

```diff
@@ -0,0 +1,44 @@
+const request = require("supertest");
+const app = require("../app.js");
+const db = require("../db/connection.js");
+const data = require("../db/data/test-data");
+const seed = require("../db/seeds/seed.js");
+
+beforeEach(() => {
+  return seed(data);
+});
+
+afterAll(() => db.end());
+
+describe("GET requests", () => {
+  it("returns a 404 if endpoint does not exist", () => {
+    return request(app).get("/api/notanendpoint").expect(404);
+  });
+  describe("/api/topics", () => {
+    it('returns with 200 status code "OK"', () => {
+      return request(app).get("/api/topics").expect(200);
+    });
+
+    it("responds with an array of topic objects", () => {
```

Comment:
this isn't really asserting what your test description is saying as it's not looking to see if the array contains objects